src/StackExchange.Redis/Enums/RedisCommand.cs

```diff
@@ -1,236 +1,465 @@
+            case RedisCommand.GETRANGE:
+            case RedisCommand.HEXISTS:
+            case RedisCommand.HGET:
+            case RedisCommand.HGETALL:
+            case RedisCommand.HKEYS:
+            case RedisCommand.HLEN:
+            case RedisCommand.HMGET:
+            case RedisCommand.HRANDFIELD:
+            case RedisCommand.HSCAN:
+            case RedisCommand.HSTRLEN:
+            case RedisCommand.HVALS:
+            case RedisCommand.INFO:
+            case RedisCommand.KEYS:
+            case RedisCommand.LASTSAVE:
+            case RedisCommand.LATENCY:
+            case RedisCommand.LINDEX:
+            case RedisCommand.LLEN:
+            case RedisCommand.LPOS:
+            case RedisCommand.LRANGE:
+            case RedisCommand.MEMORY:
+            case RedisCommand.MGET:
+            case RedisCommand.MONITOR:
+            case RedisCommand.MULTI:
+            case RedisCommand.OBJECT:
+            case RedisCommand.PFCOUNT:
+            case RedisCommand.PING:
+            case RedisCommand.PSUBSCRIBE:
+            case RedisCommand.PTTL:
+            case RedisCommand.PUBLISH:
+            case RedisCommand.PUBSUB:
+            case RedisCommand.PUNSUBSCRIBE:
+            case RedisCommand.QUIT:
+            case RedisCommand.RANDOMKEY:
+            case RedisCommand.READONLY:
+            case RedisCommand.READWRITE:
+            case RedisCommand.REPLICAOF:
+            case RedisCommand.ROLE:
+            case RedisCommand.SAVE:
+            case RedisCommand.SCAN:
+            case RedisCommand.SCARD:
+            case RedisCommand.SCRIPT:
+            case RedisCommand.SDIFF:
+            case RedisCommand.SELECT:
+            case RedisCommand.SENTINEL:
+            case RedisCommand.SHUTDOWN:
+            case RedisCommand.SINTER:
+            case RedisCommand.SINTERCARD:
+            case RedisCommand.SISMEMBER:
+            case RedisCommand.SLAVEOF:
+            case RedisCommand.SLOWLOG:
+            case RedisCommand.SMEMBERS:
+            case RedisCommand.SMISMEMBER:
+            case RedisCommand.SORT:
+            case RedisCommand.SRANDMEMBER:
+            case RedisCommand.STRLEN:
+            case RedisCommand.SUBSCRIBE:
+            case RedisCommand.SUNION:
+            case RedisCommand.SSCAN:
+            case RedisCommand.SYNC:
+            case RedisCommand.TIME:
+            case RedisCommand.TTL:
+            case RedisCommand.TYPE:
+            case RedisCommand.UNSUBSCRIBE:
+            case RedisCommand.UNWATCH:
+            case RedisCommand.WATCH:
+            case RedisCommand.XACK:
+            case RedisCommand.XADD:
+            case RedisCommand.XCLAIM:
+            case RedisCommand.XDEL:
+            case RedisCommand.XGROUP:
+            case RedisCommand.XINFO:
+            case RedisCommand.XLEN:
+            case RedisCommand.XPENDING:
+            case RedisCommand.XRANGE:
+            case RedisCommand.XREAD:
+            case RedisCommand.XREADGROUP:
```

Comment:
Dread group id guess is considered a write command

Reply:
I'm not sure on how this is handled either - the semantics of stream claiming are odd to me. For example, if you defined the consumer group on the secondary, is it valid to claim there? I have no idea if either is valid.

Reply:
meant XREADGROUP, sorry was on mobile 😆

Reply:
I've set up a sentinel, I'll take a look at the ones with open questions.

Reply:
@slorello89 I'm handling X* stuff right now - big test case incoming, working through each command!

Reply:
Can confirm this is also a write operation

Reply:
Concur - change and test added!